### DISCUSSION TITLE
[WORKERS] Definition in prod enviroment?

### DISCUSSION THREAD

**Discussion by Mat-moran** [Tecnativa/doodba#369] [Q&A] [answered]
Good morning,

I see where the workers are defined in devel and test, but **where are the defined in prod?** I do not see them anywhere in doodba-copier-template project.

Thanks an regards.

**Comment by yajo** [comment, marked as answer]
You are supposed to drop a file such as `odoo/custom/conf.d/10-performance.conf` in your scaffolding. This file should contain your performance settings, with things such as workers and limits. We don't provide defaults because correct settings depend a lot on your server.

**Reply by Mat-moran** [reply]
Thanks, it works! I found a well explained configuration here, and I used only the part relative to Tunning options, maybe helps for anyone.
https://gist.github.com/Guidoom/d5db0a76ce669b139271a528a8a2a27f#file-gistfile1-txt-L320

**Reply by yajo** [reply]
Probably the official docs help too: https://www.odoo.com/documentation/14.0/reference/cmdline.htm
